forge-gui/res/editions/Portal.txt

```diff
@@ -16,7 +16,6 @@ ScryfallCode=POR
 4 U Ardent Militia @Mike Raabe
 5 R Armageddon @John Avon
 6 C Armored Pegasus @Andrew Robinson
-6d C Armored Pegasus @Andrew Robinson
```

Comment:
So why were these removed? These are different arts: 
> Demo Game Armored Pegasus
The demo pack versions of some Portal cards had extra reminder text.